components/avatar/avatar.html

```diff
@@ -0,0 +1,117 @@
+            <div class="nav_list_container">
+                <ul class="nav_list">
+                    <li class="nav_list_icons">
+                        <a class="nav_icons" target="_blank" href="https://github.com/manojasarsa"><i
+                                class="fab fa-github fa-2x"></i></a>
+                    </li>
+                    <li class="nav_list_icons">
+                        <a class="nav_icons" target="_blank" href="https://www.linkedin.com/in/manoj-asarsa"><i
+                                class="fab fa-linkedin fa-2x"></i></a>
+                    </li>
+                    <li class="nav_list_icons">
+                        <a class="nav_icons" target="_blank" href="https://twitter.com/manoj_asarsa"><i
+                                class="fab fa-twitter fa-2x"></i></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="doc_sidebar">
+        <div class="doc_heading">Getting Started</div>
+        <div class="doc_sidebar_list">
+            <a class="doc_comp_link" href="">Installation</a>
+        </div>
+        <div class="doc_heading">Components</div>
+        <ul>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="/components/avatar/avatar.html">Avatar</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Alert</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Badge</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Button</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Card</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Image</a></li>
+            <li class="doc_sidebar_list"><a class="doc_comp_link" href="">Input</a></li>
+        </ul>
+    </div>
+
+    <!-- Main Content -->
+
+    <div class="doc_content">
+
+        <div class="avatar_container">
+            <h2 class="avatar_head">Avatar</h2>
+            <p class="avatar_desc">
+                The Avatar component is used to represent a user profile pictures on profile information page, on
+                navigation
+                bar, in blogs grid items. They usually found in circular shapes.
+            </p>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Cascadia+Code&fs=13.5px&lh=146%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fzero-ui.netlify.app%252Fcomponents%252Favatar%252Favatar.css%2522%29%253B%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fzero-ui.netlify.app%252Fcomponents%252Favatar%252Favatar.css%2522%250A%253E%250A"
+                style="width: 80vw; height: 30vh; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Good practice: Remove inline styles, it's a bad practice. It will give headache while debugging.